ansible/roles/tasks-systemd/tasks/main.yml

```diff
@@ -40,6 +40,54 @@
       maxsockets=3
       cafile=/run/secrets/tasks/npm-registry.crt
 
+- name: Create job-runner configuration
+  copy:
+    dest: /etc/job-runner.toml
+    content: |
+      [logs]
+      driver='s3'
+
+      [forge.github]
+      token = [{file="/run/secrets/webhook/.config--github-token"}]
+
+      [logs.s3]
+      # bots lib/stores.py LOG_STORE
+      url = 'https://cockpit-logs.us-east-1.linodeobjects.com/'
+      key = [{file="/run/secrets/tasks/s3-keys/cockpit-logs.us-east-1.linodeobjects.com"}]
+
+      [container]
+      command = ['podman-remote', '--url=unix:///podman.sock']
+      run-args = [
+          # resources
+          '--device=/dev/kvm',
+          '--memory=24g',
+          '--pids-limit=16384',
+          '--shm-size=1024m',
+          # qcow overlays on tmpfs
+          '--tmpfs=/tmp:size=14g',
+          '--env=TEST_OVERLAY_DIR=/tmp',
```

Comment:
@allisonkarlitskaya FYI, I just added this so that we can eventually drop https://github.com/cockpit-project/cockpituous/blob/accb1fdc6aff599ab66a4915d0a3c952a48ccddb/tasks/Containerfile#L81 which is a bit ugly, too disconnected from the other half (i.e. the `--tmpfs`), and won't work with other containers.

I also reordered these by purpose instead of primarily on type, which makes it a bit easier to maintain/understand.

Reply:
I feel like a bit more vertical whitespace would help keeping the sections visually separated from each other...